mass_mailing_switzerland/models/crm_event_compassion.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nathan Fluckiger <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, api
+
+
+class EventCompassion(models.Model):
+
+    _inherit = 'crm.event.compassion'
+
+    @api.model
+    def create(self, vals):
+        event = super(EventCompassion, self).create(vals)
+        for NewEvent in self:
```

Comment:
`variable_name` should be lowercase with underscores (code convention by odoo). The loop is not needed because create can only create one record. You can simply use `event` instead of new_event.